css/style.css

```diff
@@ -51,105 +64,243 @@ ul {
     clip: rect(0 0 0 0);
     overflow: hidden;
 }
+.header {
+    position: relative;
+    max-width: 1160px;
+    height: auto;
+    background-color: #f7e296;
+    margin-top: 40px;
+    margin-left: auto;
+    margin-right: auto;
+    padding-top: 53px;
+}
+.logo-link {
+    position: absolute;
+    left: 60px;
+    top: -21px;
+}
+.header-buttons ul {
+    margin: 0;
+}
 .header-buttons {
     display: flex;
     align-items: center;
     justify-content: space-between;
+    width: auto;
+    min-height: 50px;
+    margin-bottom: 50px;
+    margin-left: 60px;
+    margin-right: 60px;
     font-family: "Opensanslight", sans-serif;
     font-size: 15px;
 }
-.logo-link {
-    margin: 23px 0 0 199px;
-    width: 100%;
-    height: 100%;
+.user-buttons {
+    display: flex;
+    justify-content: space-between;
+    width: 500px;
```

Comment:
I think this will make your UI not responsive

Reply:
let's discuss it when we call

Reply:
changed to 50% and margin-left